app/stacks/uk-west/back-office/database.tf

```diff
@@ -44,6 +44,11 @@ resource "azurerm_mssql_server" "back_office" {
   administrator_login_password = random_password.back_office_sql_server_password.result
   minimum_tls_version          = "1.2"
 
+  azuread_administrator {

```

Comment:
There can only be a single AAD administrator defined. However, on review of the [documentation](https://docs.microsoft.com/en-us/azure/azure-sql/database/authentication-aad-overview#administrator-structure) it appears we can use an AAD group for this which is far more preferable than a single user. I'll mention it to PINS and see if a group can be created.

> The Azure AD administrator login can be an Azure AD user or an Azure AD group. When the administrator is a group account, it can be used by any group member, enabling multiple Azure AD administrators for the server

Reply:
@andylangridge - PINS have confirmed a group will be created for this purpose. We can merge the change now and update the `sql_server_azuread_administrator` `object_id` and `login_username` in a later PR once the group is provisioned, or wait until then. Although I'd prefer the former to get the URL encoding fix instantiated.